src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -144,9 +144,12 @@ private boolean shouldSab() {
 
     private boolean shouldRsb() {
         if (!main.config.LOOT.RSB.ENABLED || !target.npcInfo.extra.has(NpcExtra.USE_RSB)) return false;
-        if (usedRsb < System.currentTimeMillis() - main.config.LOOT.RSB.AMMO_REFRESH) usedRsb = System.currentTimeMillis();
+        boolean isReady = bar.findItemById("ammunition_laser_rsb-75").map(i -> i.activatable).orElse(false);
 
-        return usedRsb > System.currentTimeMillis() - 50;
+        if (isReady && usedRsb < System.currentTimeMillis() - 2000) usedRsb = System.currentTimeMillis();
```

Comment:
```suggestion
 if (isReady && usedRsb < (System.currentTimeMillis() - 2000)) usedRsb = System.currentTimeMillis();
```

Reply:
This is uneeded due to preference of operators. There can be no confusion as you can't compute a boolean and then do `bool - long`.